projects/lang-portal/frontend-flask-jinja/app2.py

```
@@ -0,0 +1,138 @@
+from flask import Flask, render_template, send_from_directory
```

Ruff (0.8.2)

`flask.send_from_directory` imported but unused

Remove unused import: `flask.send_from_directory`

(F401)